sdk/purview/purview-sharing-rest/samples-dev/manageSharesSample.ts

```diff
@@ -0,0 +1,217 @@
+  const result = await client.path("/receivedShares/{receivedShareId}", receivedShareId).get();
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to list attached received shares
+ *
+ * @summary List attached received shares
+ */
+async function getAllAttachedReceivedShares(
+  client: PurviewSharingClient,
+  storageAccountResourceId: string
+) {
+  const options: ReceivedSharesGetAllAttachedReceivedSharesParameters = {
+    queryParameters: {
+      referenceName: storageAccountResourceId,
+    },
+  };
+
+  const initialResponse = await client.path("/receivedShares/attached").get(options);
+  const pageData = paginate(client, initialResponse);
+  const result = [];
+  for await (const item of pageData) {
+    result.push(item);
+  }
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to delete a received share
+ *
+ * @summary Delete a received share
+ */
+async function deleteReceivedShare(client: PurviewSharingClient, receivedShareId: string) {
+  const initialResponse = await client
+    .path("/receivedShares/{receivedShareId}", receivedShareId)
+    .delete();
+
+  const poller = await getLongRunningPoller(client, initialResponse);
+  const result = await poller.pollUntilDone();
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to delete a sent share invitation
+ *
+ * @summary Delete a sent share invitation
+ */
+async function deleteSentShareInvitation(
+  client: PurviewSharingClient,
+  sentShareId: string,
+  sentShareInvitationId: string
+) {
+  const initialResponse = await client
+    .path(
+      "/sentShares/{sentShareId}/sentShareInvitations/{sentShareInvitationId}",
+      sentShareId,
+      sentShareInvitationId
+    )
+    .delete();
+
+  const poller = await getLongRunningPoller(client, initialResponse);
+  const result = await poller.pollUntilDone();
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to delete a sent share
+ *
+ * @summary Delete a sent share
+ */
+async function deleteSentShare(client: PurviewSharingClient, sentShareId: string) {
+  const initialResponse = await client.path("/sentShares/{sentShareId}", sentShareId).delete();
+
+  const poller = await getLongRunningPoller(client, initialResponse);
+  const result = await poller.pollUntilDone();
+  console.log(result);
+}
+
+async function main() {
+  const endpoint = process.env["ENDPOINT"] || "";
+  const senderStorageAccountResourceId = process.env["SENDER_STORAGE_ACCOUNT_RESOURCE_ID"] || "";
+  const receiverStorageAccountResourceId =
+    process.env["RECEIVER_STORAGE_ACCOUNT_RESOURCE_ID"] || "";
+
+  const credential = new DefaultAzureCredential();
+  const client = createPurviewSharingClient(endpoint, credential);
+
+  const sentShareId = "FF4A2AAE-8755-47BB-9C00-A774B5A7006E";
+  const sentShareInvitationId = "9F154FA4-93D1-426B-A908-A9CAC7192B21";
+  const receivedShareId = "0D67B9C8-A6C6-4990-9EDE-12EA059D3002";
```

Comment:
Do we need to hard code these? If so, can you please add a comment guiding users if they can copy/paste these or if they need to re-create a new set of guids?